### PR TITLE
renaming folders for exercises

### PR DESCRIPTION
**Pull request by pojeda**
take a look at the folders for rma exercises, as this section comes after nonblocking I chose a different name starting from 1X_***** @robertodr

### CODE REVIEW

**Comment by robertodr** [comment]
Hi Pedro, can you please run:
```
$ git fetch origin
$ git rebase -i origin/master
```
to rebase your branch? It's hard to follow the diff in the web interface otherwise.

**Comment by pojeda** [comment]
I pushed these changes /Pedro

**Review by robertodr — APPROVED** [review]
Thanks! I prefer to have the exercises number increasing within each day's folder (00, 01, 02, etc) but this is unimportant now. Once all exercises are settled I can do a final sweep to get the numbering consistent.
